File: Contains Duplicate.c
bool containsDuplicate(int* nums, int numsSize) {
    /*   O(n^2)  */
    
    int i,j;
    
    for (i=0;i<numsSize;i++)
        for (j=i+1;j<numsSize;j++)
            {
                if (nums[i]==nums[j])
                    return true;
            }
            
    return false;
    
    
}

File: Code/contains_duplicate.py
#!/usr/bin/env python
# encoding: utf-8


class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        res = dict()
        for i in range(len(nums)):
            if res.get(nums[i]):
                return True
            res[nums[i]] = 1
        return False
